File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    
    if (s.lower()).count("p") == (s.lower()).count("y") : 
        return True;
    elif (s.lower()).count("p") == 0 and (s.lower()).count("y") == 0 :
        return True;
    else : 
        return False;

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int pcnt = 0 ; int ycnt = 0;

        String[] array = s.toLowerCase().split("");
        
        for (int i = 0 ; i < array.length ; i++) {
            if ("p".equals(array[i])){
                pcnt++;
            } else if ("y".equals(array[i])) {
                ycnt++;
            }
        }
        if (pcnt != ycnt) {
            answer =  false;
        }
        
        return answer;
    }
}